.pre-commit-config.yaml

```diff
@@ -26,37 +26,29 @@ repos:
           - id: django-upgrade
             args: [--target-version, "3.2"]
     - repo: https://github.com/psf/black
-      rev: 21.10b0
+      rev: 21.11b1
       hooks:
           - id: black
     - repo: https://github.com/pycqa/isort
       rev: 5.10.1
       hooks:
           - id: isort
-            args:
-                [
-                    --virtual-env=venv,
-                    --profile=black,
-                    --lines-after-imports=2,
-                    --combine-as,
-                ]
+            args: [--profile=black, --lines-after-imports=2, --combine-as]
```

Comment:
@matthiask: Bei mir gibt's manchmal einen Konflikt zwischen black und isort, wenn ich das Argument `--virtual-env=venv` weglasse.

Ich kenn mich zu wenig aus um das einzuschätzen, weisst du woran das liegen könnte?

Reply:
Ich weiss nicht recht. Vielleicht müsste die Reihenfolge der Commands geändert werden? Zuerst isort ausführen, nachher erst black. Kann sein, dass isort zB zu viele Leerzeilen hinterlässt, weil es die Imports neu gruppiert, und black danach keine Freude hat.

Reply:
@matthiask: Grad noch mal in meinen Testprojekten ausprobiert und es trat jetzt nicht mehr auf.

Reply:
Super! Ich hab trotzdem die Reihenfolge noch geändert, das macht sicher Sinn.